core/runtime/src/main/java/io/quarkiverse/jberet/runtime/JBeretConfig.java

```diff
@@ -80,6 +80,26 @@
              */
             @ConfigItem(defaultValue = DataSourceUtil.DEFAULT_DATASOURCE_NAME)
             public String datasource;
+            /**
+             * Allow custom DDL file resource for JBeret tables creation;
+             * if using <b>custom table names</b> please also set <code>sql-filename</code>
+             * property to propagate table names
+             */
+            @ConfigItem(name = "ddl-filename")
+            public Optional<String> ddlFileName;
+            /**
+             * Allow custom queries to be used to query JBeret tables;
+             * this is mandatory if custom table names are used in
+             * custom DDL filename
+             */
+            @ConfigItem(name = "sql-filename")
+            public Optional<String> sqlFileName;
+            /** If present, prefix JBeret tables with this property value */
+            @ConfigItem(name = "table-prefix")
+            public Optional<String> dbTablePrefix;
+            /** If present, suffix JBeret tables with this property value */
+            @ConfigItem(name = "table-suffix")
+            public Optional<String> dbTableSuffix;
```

Comment:
I think we should align these configuration properties names with the ones used in JBeret:

https://github.com/jberet/jsr352/blob/5b2b7e60f67e14bd41588f994a3447f66fec85ac/jberet-core/src/main/java/org/jberet/repository/JdbcRepository.java#L52-L61

We can also include the defaults set by JBeret.

Reply:
About properties names I second this.
About defaults I propose to leave properties as *Optional<>* because missing values are managed by JBeret as described in this ticket (https://issues.redhat.com/browse/JBERET-298).

Reply:
Ok, sounds good. Go ahead. Thanks!

Reply:
Commited minutes ago

Reply:
Thanks. It seems that some tests are failing.

Reply:
Yes. I'm just investigating; errors are related to IT tests. 
This is the first time working with native image builds and I need some practice.
I'm sorry about that delay.

Reply:
No worries, take your time. Let me know if I can help.